sciurus17_examples_py/sciurus17_examples_py/pose_groupstate.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2025 RT Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from moveit.planning import MoveItPy
+from moveit.planning import PlanRequestParameters
+
+import rclpy
+from rclpy.logging import get_logger
+
+from sciurus17_examples_py.utils import plan_and_execute
+
+
+def main(args=None):
+    rclpy.init(args=args)
+    logger = get_logger('pose_groupstate')
+
+    # instantiate MoveItPy instance and get planning component
+    sciurus17 = MoveItPy(node_name='pose_groupstate')
+    logger.info('MoveItPy instance created')
+
+    # アーム制御用 planning component
+    arm = sciurus17.get_planning_component('two_arm_group')
+
+    arm_plan_request_params = PlanRequestParameters(
+        sciurus17,
+        'ompl_rrtc_default',
+    )
+
+    # 動作速度の調整
+    arm_plan_request_params.max_acceleration_scaling_factor = (
+        1.0  # Set 0.0 ~ 1.0
+    )
+    arm_plan_request_params.max_velocity_scaling_factor = 1.0  # Set 0.0 ~ 1.0
```

Comment:
ここでレイアウトが崩れるとやや見づらいのでこのような表記はいかがでしょうか。
```suggestion
    # Set 0.0 ~ 1.0
    arm_plan_request_params.max_acceleration_scaling_factor = 1.0
    arm_plan_request_params.max_velocity_scaling_factor = 1.0
```